src/libs/ReportUtils.js

```diff
@@ -411,18 +412,74 @@ function formatReportLastMessageText(lastMessageText) {
 }
 
 /**
- * Helper method to return a default avatar
- *
+ * Helper method to return the default avatar associated with the given login
  * @param {String} [login]
  * @returns {String}
  */
 function getDefaultAvatar(login = '') {
-    // There are 8 possible default avatars, so we choose which one this user has based
+    // There are 24 possible default avatars, so we choose which one this user has based
+    // on a simple hash of their login
+    if (!login) {
+        return Expensicons.FallbackAvatar;
+    }
+    if (login === CONST.EMAIL.CONCIERGE) {
+        return Expensicons.ConciergeAvatar;
+    }
+
+    const loginHashBucket = (Math.abs(hashCode(login.toLowerCase())) % 24) + 1;
+
+    return defaultAvatars[`Avatar${loginHashBucket}`];
+}
+
+/**
+ * Helper method to return old dot default avatar associated with login
+ *
+ * @param {String} [login]
+ * @returns {String}
+ */
+function getOldDotDefaultAvatar(login = '') {
+    // There are 8 possible old dot default avatars, so we choose which one this user has based
     // on a simple hash of their login
+    if (login === CONST.EMAIL.CONCIERGE) {
+        return CONST.CONCIERGE_ICON_URL;
+    }
     const loginHashBucket = (Math.abs(hashCode(login.toLowerCase())) % 8) + 1;
+
     return `${CONST.CLOUDFRONT_URL}/images/avatars/avatar_${loginHashBucket}.png`;
 }
 
+/**
+ * Given a user's avatar path, returns true if user doesn't have an avatar or if URL points to a default avatar
+ * @param {String} [avatarURL] - the avatar source from user's personalDetails
+ * @returns {Boolean}
+ */
+function isDefaultAvatar(avatarURL) {
+    if (_.isString(avatarURL) && (avatarURL.includes('images/avatars/avatar_') || (avatarURL.includes('images/avatars/user/default')))) {
+        return true;
+    }
+
+    // if null URL, we should also use a default avatar
+    if (!avatarURL) {
```

Comment:
nab: Why not merge both the conditions or even get rid of the if and else?

Reply:
I think it's more readable to keep it separate since the first check is so long / nested